bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooabscollection.py

```diff
@@ -33,7 +33,20 @@ def addClone(self, arg, silent=False):
         The keywords must correspond to the CmdArgs of the function.
         """
         clonedArg = self._addClone(arg, silent)
-        SetOwnership(clonedArg, False)
+        # There are two overloads of RooAbsCollection::addClone():
+        #
+        #   - RooAbsArg *addClone(const RooAbsArg& var, bool silent=false);
+        #   - void addClone(const RooAbsCollection& list, bool silent=false);
+        #
+        # In the case of the RooAbsArg overload, we need to tell Python that it
+        # doesn't own the returned pointer. That's because the function name
```

Comment:
Can you confirm this statement? `we need to tell Python that it doesn't own the returned pointer. That's because the function name contains "Clone"`

Reply:
Here is the link the the code where the check is done:
https://github.com/root-project/root/blob/master/bindings/pyroot/cppyy/CPyCppyy/src/CPPOverload.cxx#L936